### ISSUE TITLE
goToHour() wrong offset

### ISSUE THREAD

**Issue by mkruglikov** [thellmund/Android-Week-View#133]
I set minHour to 6 and when I am calling goToHour(10) it scrolls to the end of a day. Without minHour it works fine.
Probably, it's the modifiedHour variable in goToHour() method is being calculated wrong

```kotlin
val modifiedHour = min(hour, configWrapper.hoursPerDay)
var verticalOffset = configWrapper.hourHeight * modifiedHour
```

**Comment by thellmund** [comment]
Fixed in 36103a233bc1823d1e4f9b98b561d0f3ccff47d0. Will try to release a new version over the next few days.

**Comment by thellmund** [comment]
Released fix with [4.1.4](https://github.com/thellmund/Android-Week-View/releases/tag/4.1.4).
